### DISCUSSION TITLE
is the postee-config intended to be shared between the svc and UX?

### DISCUSSION THREAD

**Discussion by danekantner** [aquasecurity/postee#379] [Q&A] [answered]
is the postee-config PV intended to be shared between the svc and UX? similar question for postee-db

there is a PV for each but the access mode is set to ReadWriteOnce vs ReadWriteMany -- is the config PV used by each intended to be shared or two unrelated PVs?

**Comment by krol3** [comment, marked as answer]
Hi @danekantner the postee-UI uses the same volume as the postee server (database and cfg.yaml). I hope this information helps you to clarify the PV

>> Persistent volume
>> - postee-db: persistent storage directory `/server/database`
>> - postee-config: mount the cfg.yaml to a writable directory `/config/cfg.yaml`
>> - postee-rego-templates: mount custom rego templates
>> - postee-rego-filters: mount custom rego filters
